recipes/box2d/2.4.x/test_package/conanfile.py

```diff
@@ -1,17 +1,25 @@
 import os
-from conans import ConanFile, CMake, tools
+from conan import ConanFile
+from conan.tools.build import cross_building
+from conan.tools.cmake import CMake, cmake_layout
 
 
 class Box2DTestConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake", "cmake_find_package_multi"
+    generators = "CMakeDeps", "CMakeToolchain", "VirtualRunEnv"
+
```

Comment:
```suggestion
    test_type = "explicit"

```
Needed for Conan 1.x